src/Components/FruitCounters.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+function FruitCounter({classWrapperAttribute, counterValue, setCounterValue, classAttribute, valueAttribute}) {
+
+    function addCounterValue() {
+         return counterValue ++;
+    }
+
+    function minCounterValue() {
+         return counterValue --;
+    }
+
+    return (
+        <div className={classWrapperAttribute}>
+            <h2 className={classAttribute}>{valueAttribute}</h2>
+            <button
+                className={classAttribute}
+                onClick={() => counterValue >= 1 && setCounterValue(minCounterValue)}
```

Comment:
Als je de huidige state wil gebruiken, dan kun je beter die gebruiken die meegegeven wordt
```suggestion
                onClick={() => counterValue >= 1 && setCounterValue(curr => curr - 1)}
```
Hierdoor wordt er niet telkens een nieuwe functie aangemaakt in memory, omdat een waarde buiten de functie veranderd wordt